CryptoPkg/Library/Include/CrtLibSupport.h

```diff
@@ -437,6 +472,66 @@ strcat (
   const char  *strSource
   );
 
+int
+fflush (
+  FILE  *stream
+  );
+
+int
+ferror (
+  FILE  *fp
+  );
+
+int
+fseek (
+  FILE  *stream,
+  long  offset,
+  int   whence
+  );
+
+int
+feof (
+  FILE  *fp
+  );
+
+char *
+fgets (
+  char  *str,
+  int   n,
+  FILE  *stream
+  );
+
+int
+ftell (
+  FILE  *fp
+  );
+
+int
+fputs (
+  const char  *str,

```

Comment:
```suggestion
  const char  *restrict s,
```
Be compatible with fputs(3). I don't see other uses of `restrict` in EDK2, but I don't know why we wouldn't use it. Would need a maintainer to comment.